face_track/main.py

```diff
@@ -27,9 +27,9 @@
 
 print "Start face-tracking node"
 ft = FaceTrack()
-
+ft.do_look_at_actions()
 while not rospy.is_shutdown():
-	ft.do_look_at_actions()
+	#ft.do_look_at_actions()
```

Comment:
OK, so this change looks obviously wrong -- the whole point of protecting this is to have everything stop when rospy is no longer running.

This is also wrong in a second way: you left behind an empty spin loop -- an infinite loop that is now spinning and doing absolutely nothing at all, just wasting CPU time.

Please have other people review your code before committing changes.